docs/rules/prefer-arrow-callback.md

```diff
@@ -51,6 +53,23 @@ Examples of **correct** code for the `{ "allowNamedFunctions": true }` option:
 foo(function bar() {});
 ```
 
+### allowUnboundThis
+
+This is a `boolean` option and it is `true` by default. When set to `false` it disallows using "dynamic" `this` in callback functions where `this` is not bound using `bind` and determined by the higher-level function.
```

Comment:
We probably want the wording here to match the option name. So "When set to `false`, this option allows the use of `this` without restriction. Normally, this rule will flag the use of `this` whenever a function does not use `bind()` to specify the value of `this`."

Or some such thing.